internal/admin/dashboard/static/css/dashboard.css

```
@@ -0,0 +1,823 @@
+    height: 32px;
+    background: transparent;
+    border: none;
+    border-radius: 6px;
+    color: var(--text);
+    font-size: 12px;
+    font-family: inherit;
+    cursor: pointer;
+    transition: all 0.1s;
+}
+
+.dp-day:hover:not(.disabled):not(.other-month) {
```

Stylelint (17.2.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)